### ISSUE TITLE
IServiceCollectionExtensions extra closing parenthesis with httpMessageHandlers

### ISSUE THREAD

**Issue by EEParker** [christianhelle/refitter#205]
**Describe the bug**
Version 0.8.2 generates a new dependency injection IServiceCollectionExtensions, and seems to introduce a syntax error as shown below:

```csharp
            services
                .AddRefitClient<IBackendApi>()
                .ConfigureHttpClient(c => c.BaseAddress = baseUrl)
                .AddHttpMessageHandler<AuthorizationMessageHandler>()
                .AddHttpMessageHandler<TelemetryMessageHandler>()); // extra ) here
```

This seems to originate from this line

https://github.com/christianhelle/refitter/blob/ecca7aca4cdd217f0bfd0cb714aced54b2d08c71/src/Refitter.Core/DependencyInjectionGenerator.cs#L81

I wasn't sure if there were use cases that this worked, but this is the config that does not work, and probably has something to do with the `httpMessageHandlers` in the dependency injection section.

```json
{
    "openApiPath": "../../Services/BackendApi/wwwroot/swaggerfile.json",
    "namespace": "App.Services.Clients.BackendApiClient",
    "naming": {
      "useOpenApiTitle": false,
      "interfaceName": "BackendApiClient"
    },
    "generateContracts": true,
    "generateXmlDocCodeComments": true,
    "addAutoGeneratedHeader": false,
    "addAcceptHeaders": false,
    "returnIApiResponse": true,
    "generateOperationHeaders": true,
    "typeAccessibility": 0,
    "useCancellationTokens": false,
    "useIsoDateFormat": true,
    "multipleInterfaces": 2,
    "generateDeprecatedOperations": false,
    "operationNameTemplate": "{operationName}Async",
    "optionalParameters": true,
    "outputFolder": "./Clients",
    "includePathMatches": [
      "^(?!.*[Ii]ntegration[Ee]vent).*\/api.*"
     ],
    "dependencyInjectionSettings": {
      "httpMessageHandlers": [
          "AuthorizationMessageHandler",
          "TelemetryMessageHandler"
      ]
    }
  }
  ```
I cannot include the API spec for privacy reasons.

**Comment by christianhelle** [comment]
@EEParker Good catch!!

My (and my team's) use case is 

```json
  "dependencyInjectionSettings": {
    "baseUrl": "https://petstore3.swagger.io/api/v3",
    "usePolly": true,
    "pollyMaxRetryCount": 3,
    "firstBackoffRetryInSeconds": 0.5,
    "httpMessageHandlers": [
      "AuthorizationMessageHandler",
      "TelemetryMessageHandler"
    ]
  }
```

I never tested without using Polly! I'll fix this immediately. Sorry for the inconvenience.

**Comment by christianhelle** [comment]
@all-contributors please add @EEParker for bugs

**Comment by EEParker** [comment]
Thank you for the response and quick fix! Just FYI, our use case is [Dapr](https://dapr.io/) service invocation which has retry and backoff, so we haven't implemented Polly.  Thank you for creating and maintain this tool.

**Comment by christianhelle** [comment]
This fix is on its way to a release and will be included in v0.8.3
